### ISSUE TITLE
Add an optional link to webserver commands

### ISSUE THREAD

**Issue by Justineo** [typicode/hotel#50]
It would be extremely convenient if I can open up the web page I just served via a local webserver which I turned on using hotel.

![image](https://cloud.githubusercontent.com/assets/1726061/13562484/13be85b8-e474-11e5-8ea1-084179e2a444.png)

**Comment by typicode** [comment]
Hi @Justineo,

I'm not sure to understand how it would be different from clicking on `phoebe-web` or what `link to local URL` should do?

Currently if you click on `phoebe-web` you'll be redirected to `http://phoebe-web.dev` or `http://localhost:<some_port>`.

If you can give me an example of what it should do?

**Comment by Justineo** [comment]
> Currently if you click on phoebe-web you'll be redirected to http://phoebe-web.dev or http://localhost:<some_port>.

Yes I'm aware of this and in my case the entry URL of my web site may not be `index.html` or `http://phoebe-web.dev`, it may be something like `http://phoebe-web.dev/entry/dashboard.html?debug=true`.

**Comment by typicode** [comment]
Ok thank you, I understand a little better. So `link to local URL` would be some configuration option (for example: `hotel add ... --entry /entry/dashboard.html?debug=true`)?

**Comment by Justineo** [comment]
Yes. That's exactly what I mean. :)

**Comment by Justineo** [comment]
I dug a little bit into the code and found that because `respawn`'s `Monitor.prototype.toJSON` ignores custom properties it will be more complicated than I think to support this feature gracefully now :(

**Comment by Justineo** [comment]
Hi, I made a video to show the feature: https://youtu.be/CLCI43jeX2E
